test/inductor/test_triton_extension_backend.py

```diff
@@ -65,6 +65,7 @@ def mock_triton_hash_with_backend(*args, **kwargs):
 
 
 @unittest.skipIf(IS_FBCODE, "cpp_extension doesn't work in fbcode right now")
+@test_torchinductor.skip_if_cpp_wrapper("Tricky to fix, will return to later")
```

Comment:
@desertfire This test may take some substantive work to support. I'm going to address it in a follow-up PR.

Reply:
@desertfire After testing, I don't think this will work until we can support `triton-cpu` as a codegen backend for `CppWrapperCpu`, so I've updated the skip message to say that. Not sure there's much to fix here in the test.